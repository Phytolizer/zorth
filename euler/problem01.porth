include "std.porth"

0 3 while dup 1000 < do
  dup  3 mod 0 =
  over 5 mod 0 =
  bor if
     swap over + swap
  end

  1 +
end drop
print
