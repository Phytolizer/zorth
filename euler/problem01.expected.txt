: returncode 0
: stdout 7
233168

: stderr 0

